src/core_modules/capture-core/components/WidgetProfile/WidgetProfile.component.js

```diff
@@ -101,7 +105,11 @@ const WidgetProfilePlain = ({
             return <span>{i18n.t('Profile widget could not be loaded. Please try again later')}</span>;
         }
 
-        return <FlatList dataTest="profile-widget-flatlist" list={displayInListAttributes} />;
+        return (
+            <div className={classes.container}>
+                <FlatList dataTest="profile-widget-flatlist" list={displayInListAttributes} />
+            </div>
```

Comment:
Putting the layout styles on the `container`, rather than on the `FlatList`. This helps `FlatList` remain a flexible component that doesn't have it's own positioning.